webpack.config.js

```diff
@@ -108,20 +109,22 @@ const base = {
                   browsers: ['last 1 version', 'not dead', '> 0.2%']
                 }
               }
-            ],
-            '@babel/preset-react',
-            '@babel/preset-typescript'
+            ]
           ].map(resolve)
         }
+      },
+      {
+        test: /\.tsx?$/,
+        use: [
+          { loader: 'ts-loader' },
+          { loader: 'angular2-template-loader' },
+        ],
+        exclude: /(node_modules|bower_components|\.embark[\\/]versions|(\.(spec|e2e)\.ts$))/
```

Comment:
Looks like this is problematic... Embark has this "magic" import path `Embark/contracts/SomeContract`, for contracts that are generated inside `.embark`. Having this excluded makes it impossible to load contract instances in the app as described here: https://embark.status.im/docs/contracts_javascript.html

Reply:
I don't think it's problematic because this loader rule targets `*.tsx?` files only and generated contrats are `*.js` files in `./embark/contracts`.

In addition, the initial loader rule which targets `*.jsx?` files already excludes `.embark` folder and it's not problematic.

Finally, I tried with SimpleStorage example and it's works great: I was able to import and log its content. Below the output.

**Can we resolve this discussion?**

![Uploading embark-angular-simple-storage.png…](Embark Angular template: output SimpleStorage contract)

Reply:
Okay I can take another look... I tried it and TS wasn't able to resolve contracts from `Embark/contracts/`.  But chances are also high that this is related to 4.0.0-alpha.

Can you test with latest `master` and confirm?